docs/src/content/docs/api/tools.mdx

```diff
@@ -0,0 +1,628 @@
+---
+title: "Tools"
+sidebar:
+  order: 30
+---
+import { Badge, Tabs, TabItem } from '@astrojs/starlight/components';
+
+Tools in Ultravox allow you to extend the capabilities of your AI assistant by giving it access to external APIs and services. With tools, your AI can perform actions like fetching real-time data, interacting with databases, or integrating with third-party services, all while maintaining a natural conversation flow.
+
+### Two Types of Tools
+Ultravox supports durable and temporary tools. There is no functional difference between them. Durable tools are persisted and can be reused across calls or applications. Temporary tools are created each time you create a new Call.
+
+Iteration is faster when using temporary tools. Durable tools shine once you have things dialed in, when you want to share tools across multiple applications, or if you have split responsibilities on the team.
+
+### Creating Tools
+Temporary tools are defined and passed in the request body of the [`Create Call`](../calls/#create-call) endpoint.
+
+You create durable tools either by uploading an OpenAPI spec or via the request body in the [`Create Tool`](./#create-tool) endpoint. Your OpenAPI spec must be either `json` or `yaml` format.
+
+## List Tools
+<Badge text="GET /tools" variant="note" size="large" />
+
+Lists all available tools. API key scoped.
+
+<Tabs>
+  <TabItem label="Parameters">
+    <table class="w-full">
+      <tr class="w-full">
+        <th class="w-1/12"></th>
+        <th class="w-11/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">cursor</td>
+        <td>Pagination cursor.</td>
+      </tr>
+    </table>
+  </TabItem>
+  <TabItem label="Request Body">
+    <table class="w-full">
+      <tr class="w-full">
+        <th class="w-1/12"></th>
+        <th class="w-1/12"></th>
+        <th class="w-10/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">None</td>
+        <td></td>
+        <td></td>
+      </tr>
+    </table>
+  </TabItem>
+  <TabItem label="Response">
+    <table>
+      <tr>
+        <th class="w-1/12"></th>
+        <th class="w-1/12"></th>
+        <th class="w-10/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">next</td>
+        <td>string</td>
+        <td>URL for the next page of results.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">previous</td>
+        <td>string</td>
+        <td>URL for the previous page of results.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">results</td>
+        <td>array</td>
+        <td>Array of [Tool](./#tool-definition-schema) objects.</td>
```

Comment:
The objects in this array have toolId, name, etc. in addition to their definition.  I think both this statement and the one on line 111 could be considered correct in isolation, but it feels odd for them to point to the same place given that the objects being described are different (without a bit of explanation in the link target at least).

Reply:
ok thx. you are totally right. missed spelling out the various object params. beefing up the info here to be more clear.